src/features/conversations/components/ConversationMessagesLoader.tsx

```diff
@@ -0,0 +1,71 @@
+import Box from "@mui/material/Box";
+import Skeleton from "@mui/material/Skeleton";
+import { useMemo } from "react";
+import { colors } from "styles/colors";
+
+export const ConversationMessagesLoader = () => {
+  return (
+    <Box
+      sx={{
+        height: "calc(100% - 50px)",
+        display: "flex",
+        flexDirection: "column",
+        marginLeft: (theme) => theme.spacing(1),
+        marginRight: (theme) => theme.spacing(3),
+        marginBottom: (theme) => theme.spacing(1),
+        overflowY: "hidden",
+      }}
+    >
+      <MessageSkeleton />
+      <MessageSkeleton />
+      <MessageSkeleton />
+      <MessageSkeleton />
+      <MessageSkeleton />
+      <MessageSkeleton />
+      <MessageSkeleton />
```

Comment:
Maybe we can shorten this to something like this?
```suggestion
      {[...Array(7)].map((_e, i) => <MessageSkeleton key={i} />)}
```